src/components/atoms/OcContextualHelper/OcContextualHelper.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div class="oc-contextual-helper">
+    <oc-button :id="`${buttonId}`" appearance="raw">
```

Comment:
No need to do string interpolation here since the computed prop already returns a string (?) ;)